### PR TITLE
MWC7K 2018 wiki article

### PR DESCRIPTION
**Pull request by juankristal**
### Description

Added MWC7K 2018 wiki article and logo.

### Status

Finished; Pending Review

### CODE REVIEW

**Comment by juankristal** [comment]
Please don't punch me for a billion commits :)

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+## Ruleset
+### Tournament Rules
+
+1. The osu!mania 7K World Cup is a country-based team tournament, played on the osu!mania game mode.
+   - **While this competition is planned as a 2 versus 2 setup, this might change depending on the amount of incoming registrations.**
+
+2. Beatmap scoring is based on Score V2.
+
+3. The beatmaps for each round will be announced by the mapset selector in advance on the Sunday before the actual matches take place. Only these will be used during the respective matches.
```
Comment (PopeGadget) [inline wiki/Tournaments/MWC/2018/7K/en.md]: _mapset_ selector
is this intentional?

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+   - The Group Stage mappool will be announced after the drawings.
+   - One beatmap will be given as a tiebreaker beatmap. This beatmap will only be played in case of a tie.
+
+4. Match schedule will be settled by the Tournament Management (see below).
+5. If no staff or referee is available, the match will be postponed.
+6. Failed players scores do not get added to the team score.
```
Comment (PopeGadget) [inline wiki/Tournaments/MWC/2018/7K/en.md]: apostrophe after _players_, i.e "players' scores"

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+   - Reviving and surviving during a beatmap gets considered as passing it.
```
Comment (PopeGadget) [inline wiki/Tournaments/MWC/2018/7K/en.md]: replace _gets_ with "is"

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+7. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.
+8. If the beatmap ends in a draw, the game will be nullified.
```
Comment (PopeGadget) [inline wiki/Tournaments/MWC/2018/7K/en.md]: I think an explanation on what a game nullification actually means would be preferred

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+13. Lag is not a valid reason to nullify a beatmap.
+14. All players are supposed to keep the match running fluent and without delays. Excessive match delays coming from the player's side can be issued with penalties.
+15. If a player disconnects between the beatmaps and the team can not provide an exchange, the match can be delayed 10 minutes at maximum.
+16. All players and referees must to be treated with respect. Instructions of the referees and tournament management is to be followed. Decisions labeled as final are not to be objected.
+17. Disrupting the match by foul play, picking inappropriate warmup beatmaps (see below), insulting and provoking other players or referees, delaying the match or other deliberate inappropriate misbehavior is strictly prohibited.
+18. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
+    - Breaking the chat rules results in a silence. Silenced players can not participate at multiplayer matches and must be exchanged for the time being.
+
+19. In Group stage, 'Win by default' will be considered as win by 4:0, +1.0 score difference ratio.
```
Comment (PopeGadget) [inline wiki/Tournaments/MWC/2018/7K/en.md]: replace _In Group stage,_ with "In the Group Stage,"
replace  _'Win by default'_ with "a 'Win by Default'"
add an "a" between _as_ and _win_, i.e "as a win"

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+20. Unexpected incidences are handled by the tournament management. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
+21. Penalties for violating the tournament rules can be:
+    - Exclusion of specific players for one beatmap
+    - Exclusion of specific players for an entire match
+    - Declaring the match as Lost by Default
```
Comment (PopeGadget) [inline wiki/Tournaments/MWC/2018/7K/en.md]: add apostrophes around _Lost by Default_ for syntax consistency with the "'Win by Default'" quote above

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+    - Disqualification from the entire tournament
+    - Disqualification from the current and future official tournaments until appealed
```
Comment (PopeGadget) [inline wiki/Tournaments/MWC/2018/7K/en.md]: full stops after lines 102-106 for syntax consistency

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+   - Most beatmaps won.
+   - **Winner of the match played previously between the tied teams.**
+   - In the event of a triple tie:
+     - Have higher `∑{(total score difference) / (maximum score)}`.
+     - Winner of the rematch.
+
+4. The top 2 teams of each group will move on to the Knock-Out Stages.
+   -  This may change with the actual Group Stage setup.
+
+5. Following stages are Double Elimination Stages. This means that the winner moves to the next stage and the losing team gets moved to the Loser bracket.
```
Comment (PopeGadget) [inline wiki/Tournaments/MWC/2018/7K/en.md]: add an "on" between _moves_ and _to_, i.e "moves on to"
add an "s" and an apostrophe after Loser, i.e "Losers' bracket"

File: wiki/Tournaments/MWC/2018/7K/en.md [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+| Finals | 2018-03-10/2018-03-11 |
+| Grand Finals | 2018-03-17/2018-03-18 |
+
+## Prizes
+
+In every world cup, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. It is possible that these items change with every installment of the World Cups.
+
+| Placing | Prize(s) |
+| --- | :--- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile badge, "osu! Champion" user title for one year |
```
Comment (Ephemeralis) [inline wiki/Tournaments/MWC/2018/7K/en.md]: shouldn't this be osu!mania Champion, or am I mistaken?

Comment (juankristal) [reply wiki/Tournaments/MWC/2018/7K/en.md]: You are correct, I actually changed that in the announcement but forgot to do so in the wiki article oops.

**Pull request merged**